config/locales/de.yml

```diff
@@ -904,6 +904,8 @@ de:
         reload: Zusammenfassung neu laden
         with_extra_charge: 'mit Aufschlag:'
         without_extra_charge: 'ohne Aufschlag:'
+      edit_note:
+        title: Notitie bewerken
```

Comment:
This looks like the Dutch translation :) Should be `Notiz bearbeiten`,

Reply:
ha.  ok.  done